diff --git t/resty/sanity.t t/resty/sanity.t
index fa35c63..9bf3776 100644
--- t/resty/sanity.t
+++ t/resty/sanity.t
@@ -52,7 +52,11 @@ return "hello world"
 === TEST 4: bad file name
 --- opts: 'print("hello")'
 --- err
-Lua input file print("hello") not found.
+error: Lua input file print("hello") not found.
+
+Usage: rusty-cli [OPTIONS] [lua-file] [args]...
+
+For more information, try '--help'.
 --- ret: 2
 
 
